bridges/snowbridge/primitives/outbound-queue/src/v2/converter/convert.rs

```diff
@@ -105,14 +104,8 @@ where
 			_ => None,
 		}
 		.ok_or(AssetResolutionFailed)?;
-		let weth_address = match_expression!(
-			fee_asset_id.0.unpack(),
-			(0, [AccountKey20 { network, key }])
-				if self.network_matches(network),
-			H160(*key)
-		)
-		.ok_or(FeeAssetResolutionFailed)?;
-		ensure!(weth_address == WETHAddress::get(), InvalidFeeAsset);
+		// Check the fee asset is Ether
+		ensure!(fee_asset_id.0 == Here.into(), InvalidFeeAsset);
```

Comment:
Why is the address `Here` (BH)? Should it not be Ethereum?

Reply:
It's reanchored in view of the destination(i.e. Ethereum) which is `Here`.